src/API/Data/UnitsRepository.cs

```diff
@@ -128,6 +128,13 @@ public class UnitsRepository : DataRepository
                 //Remove UnitMembers for unit
                 db.UnitMembers.RemoveRange(unitMembers);
 
+                var supportRelationships = db.SupportRelationships
+                    .Include(r => r.Unit)
+                    .Where(r => r.UnitId == unit.Id);
+                
+                //Remove SupportRelationships for unit
+                db.SupportRelationships.RemoveRange(supportRelationships);
```

Comment:
We've got something similar going on when we delete Units.  It'd be nice if we came up with a way to log these relationships with the entity being deleted.